html/janus.js

```diff
@@ -2238,4 +2275,18 @@ function Janus(gatewayCallbacks) {
 			return true;	// Default is true
 		return (trickle === true);
 	}
+
+	function isScreenshare(media) {
+		return (
+			media.video === 'screen' ||
+			media.video === 'window' ||
+			media.video === 'tab' ||
+			(
+				typeof media.video === "object" &&
+				media.length &&
+				(( media.indexOf('screen') + media.indexOf('window') + media.indexOf('tab')) > -3)
```

Comment:
I think this was supposed to check media.video.length and media.video.indexOf() so that you could pass in an array of "window/tab/screen".  However even if you did pass in an array here I don't think it would change anything.  So I'm going to remove this conditional